frontend/components/CardAPI.vue

```diff
@@ -9,12 +9,14 @@
           style="padding: 3px"
           class="text-wrapper card-subtitle mb-2 pe-2 category-container text-body-secondary"
         >
+        <NuxtLink :to="'/categories/'+subtitle" class="subtitle-link">
```

Comment:
This can be changed to:
```typescript
<NuxtLink :to=`/categories/${subtitle}` class="subtitle-link">
```
The string interpolation is a best practice instead string concatenation 😄 
Can you change it? Thanks!